executor/show.go

```diff
@@ -1065,6 +1066,12 @@ func ConstructResultOfShowCreateTable(ctx sessionctx.Context, tableInfo *model.T
 		fmt.Fprintf(buf, " ON COMMIT DELETE ROWS")
 	}
 
+	if tableInfo.PlacementPolicyRef != nil {
+		fmt.Fprintf(buf, " /*T![placement] PLACEMENT POLICY=\"%s\" */", tableInfo.PlacementPolicyRef.Name.String())
```

Comment:
how about use `` to clarify a identifier object?

Reply:
You mean `" /*T![placement] PLACEMENT POLICY="%s" */"` ?

Reply:
" /*T![placement] PLACEMENT POLICY=`%s` */"

Reply:
Got

Reply:
It seems opts also shoud change like that: 
```
CREATE TABLE `t3` (
  `a` int(11) DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin /*T![placement] PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-east-2,us-west-1,us-west-2" */;
```